packages/app/src/providers/utils.ts

```diff
@@ -102,3 +102,13 @@ export function createAxiosProgressHandler(
     })
   );
 }
+
+export class AbortError extends Error {
+  public constructor(abortSignal?: AbortSignal, cause?: unknown) {
+    const message =
```

Comment:
Shouldn't the message be always `CANCELLED_BY_USER` ?

Reply:
We call `valuesStore.abortAll()` when switching entity and visualization to avoid potentially piling on unnecessary requests.

Reply:
Technically, when this happens, we evict those same abort errors from the store right away so the `ErrorFallback` is never shown – e.g.

```ts
  function onSelectPath(path: string) {
    setSelectedPath(path);
    valuesStore.abortAll('entity changed');
    valuesStore.evictErrors();
  }
```

This means I could potentially remove the check for `message === CANCELLED_BY_USER` in `ErrorFallback` and only check for `error instanceof AbortError`. But I still think it's important to store an abort reason in the `AbortError` for debugging purposes.

Reply:
I've gone ahead and simplified the check in `ErrorFallback` (and also removed the `CANCELLED_BY_USER` constant). Even if "Request cancelled - Retry?" were to be shown for other abort reasons, it would still make sense.

As it turns out, I've found a race condition: abort errors caused by switching entity or visualization are not always properly evicted from the store. I've got a solution in mind.

Reply:
> I've gone ahead and simplified the check in ErrorFallback (and also removed the CANCELLED_BY_USER constant). 

Great, that's indeed simpler I think :+1: